src/components/ValueSet/ValueSetList.tsx

```diff
@@ -269,18 +267,16 @@ const RenderTable = ({
                   </div>
                 </TableCell>
                 <TableCell className="whitespace-nowrap px-6 py-4 text-sm">
-                  {!valueset.is_system_defined && (
-                    <Button
-                      variant="outline"
-                      size="sm"
-                      onClick={() =>
-                        navigate(`/admin/valuesets/${valueset.slug}/edit`)
-                      }
-                    >
-                      <Pencil className="size-4 mr-0" />
-                      {t("edit")}
-                    </Button>
-                  )}
```

Comment:
Hmm 🤔  Since it maybe worth adjusting the verbiage from edit to preview for system defined value sets.